custom_components/tahoma/__init__.py

```diff
@@ -144,11 +145,25 @@ async def handle_execute_command(call):
         """Handle execute command service."""
         entity_registry = await hass.helpers.entity_registry.async_get_registry()
         entity = entity_registry.entities.get(call.data.get("entity_id"))
-        await tahoma_coordinator.client.execute_command(
-            entity.unique_id,
-            Command(call.data.get("command"), call.data.get("args")),
-            "Home Assistant Service",
-        )
+        try:
+            await tahoma_coordinator.client.execute_command(
+                entity.unique_id,
+                Command(call.data.get("command"), call.data.get("args")),
+                "Home Assistant Service",
+            )
+        except Exception as e:
+            if e.args[0].startswith("No such command"):
```

Comment:
Would be nice to have a dedicated exception instead of parsing the message.

Reply:
I know, but the client only throws Exception there

Reply:
https://github.com/iMicknl/python-tahoma-api/blob/282ab59d56172d799ffeebb89fbfd07c7dd8fe57/pyhoma/client.py#L334

Reply:
https://github.com/iMicknl/python-tahoma-api/pull/54